app/frontend/src/components/form/OrgBookSearch.vue

```diff
@@ -77,15 +78,14 @@ export default {
       this.isLoading = true;
 
       // Lazily load results
-      fetch(
+      axios.get(
```

Comment:
Ideally we'll want to move this out into its own service component, but this is good enough for now as it doesn't require Bearer token injection.

Reply:
Yeah same deal with geo city lookup. Will look at it in that PR. Good call.